README.md

```diff
@@ -0,0 +1,13 @@
+**MEI 3 to 5**
+=====================================
+
+This repository contains a collection of scripts designed to update MEI 3 files to MEI 5. The current implementation of this repository only supports upgrading MEI files from the Liber Usualis. Future developments may extend this functionality to support additional MEI files.
+
+**Usage**
+-----
+
+1. If you do not have it already, install lxml 5.0 using `pip install lxml`
+2. Clone the repository and download the scripts in this repository into the same directory as your MEI files
+3. Run the `liberbatch.py` script
```

Comment:
Looking at `liberbatch.py`: won't this look in `C:\Users\cole_\Downloads\...` on everyone's machine, which probably won't exist except for yours? 

This point is out of scope of this PR, so I've made another issue.